File: dataset_reduzido/codenet/Java/Code/s575985145.java
import java.util.Scanner;

public class Main{
	
	public static void main(String [] args) {
		Scanner scan = new Scanner(System.in);
		int size = scan.nextInt();
		int[][] graph = new int[size][size];
		
		int id = 0;
		int cols = 0;
		int[] row = new int[0];
		for(int i = 0; i < size; i++) {
			id = scan.nextInt();
			cols = scan.nextInt();
			row = new int[size];
			for(int j = 0; j < cols; j++)
				row[scan.nextInt() - 1] = 1;
			graph[id - 1] = row;
		}
		
		for(int i = 0; i < graph.length; i++) {
			for(int j = 0; j < graph[i].length; j++){
				System.out.print(graph[i][j]);
				if(j < graph[i].length - 1)
					System.out.print(" ");
			}
			System.out.println();
		}
	}
}

File: output/LLaMa/codenet/Java/Python/s575985145.py
def main():
    size = int(input())
    graph = [[0]*size for _ in range(size)]

    id = 0
    cols = 0
    for i in range(size):
        id = int(input())
        cols = int(input())
        row = [0] * size
        for j in range(cols):
            index = int(input()) - 1
            row[index] = 1
        graph[id - 1] = row

    for i in range(size):
        for j in range(size):
            print(graph[i][j], end=' ')
        print()

if __name__ == "__main__":
    main()